test/integration/taxon_browsing_test.rb

```diff
@@ -216,9 +228,34 @@ def and_i_can_see_links_to_the_child_taxons_in_an_accordion
     end
   end
 
+  def and_i_can_see_the_overview_section_in_the_accordion
+    subsection = first('.subsection')
+
+    assert subsection.has_selector?('.subsection-title', text: /overview/i)
+    assert subsection.has_selector?(
+      '.subsection-content .subsection-list-item a',
+      count: 2
+    )
+  end
+
+  def then_there_is_no_overview_section_in_the_accordion
+    subsection = first('.subsection')
+
+    assert_equal(
+      false,
+      subsection.has_selector?('.subsection-title', text: /overview/i)
+    )
+  end
+
+  def and_there_is_no_content_presented_after_the_accordion
```

Comment:
Is `parent-topic-contents` used elsewhere in a way that could cause a regression? I'm wondering if this step is really necessary - would we have added it if the first iteration had put the content into the accordion?